src/lib/plugin.js

```diff
@@ -95,26 +97,16 @@ module.exports = class PluginVirtual extends EventEmitter2 {
       authToken: this._authToken
     })
 
-    this._rpc.addMethod('send_message', this._handleMessage)
```

Comment:
Would there be a way to add send_request but not remove send_message just yet? Then we can already merge this, and the upgrade path will be less painful later on.